cdap-distributions/src/hdinsight/pkg/install.sh

```diff
@@ -22,9 +22,9 @@
 # The git branch to clone
 CDAP_BRANCH='release/4.1'
 # Optional tag to checkout - All released versions of this script should set this
-CDAP_TAG='v4.1.0'
+CDAP_TAG='v4.1.1'
```

Comment:
Use `CDAP_TAG=''` here, too. Otherwise, you'll break http://builds.cask.co/browse/CDAP-HPT1 builds.